virtio-devices/src/balloon.rs

```diff
@@ -138,6 +179,49 @@ impl BalloonEpollHandler {
     }
 
     fn release_memory_range(
+        &mut self,
+        memory: &GuestMemoryMmap,
+        range_base: GuestAddress,
+        range_len: usize,
+    ) -> result::Result<(), Error> {
+        let page_size: u64 = get_page_size();
+        if page_size == 1 << VIRTIO_BALLOON_PFN_SHIFT {
+            return Self::discard_memory_region(memory, range_base, range_len);
+        }
+
+        self.pbp = Some(PartiallyBalloonedPage::new());
+        let mut addr = range_base.0;
+        let mut len = range_len as i64;
+        while len > 0 {
+            if !self.pbp.as_ref().unwrap().pfn_match(addr) {
+                self.pbp.as_mut().unwrap().reset();
```

Comment:
I think the comments for this code path should be above this line.

Reply:
Yes, will fix it.